File: packages/insomnia-httpsnippet/test/fixtures/output/ocaml/cohttp/query.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value" in

Client.call `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/fetch/query.js
const fetchOptions = {
  mode: 'cors',
  method: 'GET',
  headers: {},
};

fetch('http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value', fetchOptions)
  .then(response => response.json())
  .then(data => console.log(data));
